src/clr-angular/emphasis/_labels.clarity.scss

```diff
@@ -5,12 +5,12 @@
 // TODO: Split badges and labels into separate files.
 
 @mixin generateLabelStyles($color: gray, $alias: 1) {
-  &.label-#{$color},
+  &.label-#{'' + $color},
```

Comment:
Hard error with this color variable in the dart sass version.

`You probably don't mean to use the color value orange in interpolation here.
It may end up represented as orange, which will likely produce invalid CSS.
Always quote color names when using them as strings or map keys (for example, "orange").
If you really want to use the color value here, use '"" + $color'.`